.nvmrc

```diff
@@ -1 +1 @@
-lts/gallium
+18
```

Comment:
```suggestion
lts/hydrogen
```

Reply:
Arnoud requested an version number instead of a version name (or at least prevent us from having to map to versions). He mentioned this would for sure work in the CI build.
Looking at it, that seems to also work with `.nvmrc`, so not sure this is still the case.

Either way, why would you prefer a name vs a number?

Reply:
Make sense, right